src/plugins/touch/touch.js

```diff
@@ -25,6 +25,7 @@
 
     document.addEventListener( "touchstart", function( event ) {
         lastX = startX = event.touches[ 0 ].clientX;
+        event.preventDefault();
```

Comment:
Is this one necessary? Seems like it could prevent legitimate touchstart events reaching their intended destinations.

If it is necessary, please add a comment to explain exactly what problem this fixes, on which browsers, etc.

Reply:
In fact, I know nothing about it.
Well, I changed some code in my copy to make it works on old browsers. (etg. for of -> for in) I test it on Via (an browser using Android's WebView, for old phones) on Android 4.4, but I can't swipe. 
It is said that it will auto 'touchcancel' after 'touchstart' with scrolling.So I add this. 
( https://m.2cto.com/kf/201411/348858.html )...

Reply:
Maybe new browsers needn't event.preventDefault(); I haven't tested yet.